extensions/guacamole-auth-radius/src/main/resources/templates/radiusResponseField.html

```diff
@@ -1 +1,6 @@
-<input type="password" ng-model="model" ng-trim="false" autocorrect="off" autocapitalize="off" placeholder="{{radiusPlaceholder}}" />
+<input
+    type="password"
```

Comment:
If moving to a simple `<input type="password" ...>`, any reason to continue using the custom `RadiusChallengeResponseField` instead of the standard `PasswordField`?

Reply:
Yeah, that makes sense.  Removed.